packages/blockchain-wallet-v4-frontend/src/data/components/exchange/exchange.sagas.js

```diff
@@ -320,6 +320,7 @@ export default ({ api, coreSagas, networks }) => {
       yield put(
         A.setSourceFee({
```

Comment:
setSourceFee is done 3 times, I guess we should update mempoolFees with some value everywhere